### PR TITLE
Turing info sesh

### PR DESCRIPTION
**Pull request by KatBrandt**

### CODE REVIEW

File: info-turing/index.md [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+We are so glad you are here! To get set up...
+1. Please change your Zoom name to first name, last initial, and pronouns, like this: _Kayla G. (she/her)_. Not sure how to do that? Hover over your image in the Zoom gallery, click the three dots, and select Rename.
+2. Heads up! We will ask you to briefly introduce yourself in a minute.
+
+## Goals
+
+The goal of this session is to give you a general overview of Turing:
+- what we teach and why
+- what it's like to be a student here
+- what support we offer for the job hunt
```
Comment (kaylagordon) [inline info-turing/index.md]: Capitalize the first letter of each bullet point

File: info-turing/index.md [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+- **Zoom Chat**: Enter your answer in the Zoom chat - How much do you know about Turing, the school? `Absolutely nothing`, `I have a vague idea`, `I know all about it!`, or something else!
+- **Unmute**: Introduce yourself! Please share your name, pronouns, location, and what you're hoping to get out of this session.
+
+## Workshop Overview:
+- [Who Is Turing?](#who-is-turing?)
+- [What We Teach](#what-we-teach)
+- [How We Teach](#how-we-teach)
+- [Professional Development and Job Hunt Support](#professional-development-and-job-hunt-support)
+- [Your Questions Here](#your-questions-here)
+- [Next Steps](#next-steps)
```
Comment (kaylagordon) [inline info-turing/index.md:41]: Love the layout! Having bookmarks to easily navigate the page, while having everyone on one page is GREAT

**Review by kaylagordon — COMMENTED** [review]
Looks great to me! I wonder if adding a few photos would help break up the text a bit. We could include the same pics we used for EB events.

File: info-turing/index.md (hunk 1 above)
Comment (KatBrandt) [reply info-turing/index.md]: Good catch! Updated

**Pull request merged**